### PR TITLE
Fix: Remove unused method

### PR DESCRIPTION
**Pull request by localheinz**
### What is the reason for this PR?

- [x] removes an unused method

### Author's checklist

- [x] Follow the [Contribution Guide](https://github.com/FakerPHP/Faker/blob/main/.github/CONTRIBUTING.md)

### Review checklist

- [x] All checks have passed
- [ ] Changes are approved by maintainer

### CODE REVIEW

File: src/Faker/Container/Container.php [hunk 1]
```diff
@@ -134,12 +134,4 @@ public function has($id): bool
...
         return array_key_exists($id, $this->definitions);
     }
-
-    /**
-     * Get the bindings between Extension interfaces and implementations.
-     */
-    public function getDefinitions(): array
-    {
-        return $this->definitions;
-    }
```
Comment (localheinz) [inline src/Faker/Container/Container.php:144]: Not sure what the intention with this method was, but it is currently unused.

File: src/Faker/Container/ContainerInterface.php [hunk 2]
```diff
@@ -6,8 +6,4 @@
 
 interface ContainerInterface extends BaseContainerInterface
 {
-    /**
-     * Get the bindings between Extension interfaces and implementations.
-     */
-    public function getDefinitions(): array;
```
Comment (localheinz) [inline src/Faker/Container/ContainerInterface.php:12]: Given that the `ContainerInterface` interface would now be empty, perhaps we should remove it?

**Review by bram-pkg — APPROVED** [review]

File: src/Faker/Container/ContainerInterface.php (hunk 2 above)
Comment (bram-pkg) [reply src/Faker/Container/ContainerInterface.php:12]: No harm is maintaining our own extension of it, in case it's needed.

**Comment by localheinz** [comment]
Thank you, @bram-pkg!

**Pull request merged**